api/src/database/database.py

```diff
@@ -77,82 +93,44 @@ def __init__(self, echo_sql=False):
         with Database.lock:
             if Database.initialized:
                 return
+
             Database.initialized = True
             load_dotenv()
-            self.engine = None
+            self.logger = logging.getLogger(__name__)
             self.connection_attempts = 0
-            self.SQLALCHEMY_DATABASE_URL = os.getenv("FEEDS_DATABASE_URL")
-            self.echo_sql = echo_sql
-            self.start_session()
+            database_url = os.getenv("FEEDS_DATABASE_URL")
+            if database_url is None:
+                raise Exception("Database URL not provided.")
+            self.engine = create_engine(database_url, echo=echo_sql, pool_size=10, max_overflow=0)
```

Comment:
[suggestion]: have the pool size as an os variable, so we can play with the size without the need of re-package the python function

Reply:
Done